synapse/storage/schema/main/delta/20/pushers.py

```diff
@@ -81,7 +81,3 @@ def run_create(cur, database_engine, *args, **kwargs):
     cur.execute("DROP TABLE pushers")
     cur.execute("ALTER TABLE pushers2 RENAME TO pushers")
     logger.info("Moved %d pushers to new table", count)
-
-
-def run_upgrade(*args, **kwargs):
```

Comment:
This is safe to remove, because we do a hasattr check: https://github.com/matrix-org/synapse/blob/9bb2eac71962970d02842bca441f4bcdbbf93a11/synapse/storage/prepare_database.py#L510

Reply:
I should have called that out, sorry. 👍